### ISSUE TITLE
"mixins.scss" and "variables.scss" files aren't exist in the "sass" folder

### ISSUE THREAD

**Issue by AlexandrChazov** [6pac/SlickGrid#1003]
### Describe the bug

Hello, I use "slickgrid" in "react" and unstalled it with "npm"
When I try to import styles following the [migration guide](https://github.com/6pac/SlickGrid/wiki/Major-version-5.0-%E2%80%90-ES6-ESM-and-TypeScript-Support#:~:text=%40import%20%27slickgrid/dist/styles/sass/slick%2Dalpine%2Dtheme.scss%27%3B)
`@import 'node_modules/slickgrid/dist/styles/sass/slick-alpine-theme.scss';`
I receive SassError: "Can't find stylesheet to import".
After opening file "node_modules/slickgrid/dist/styles/sass/slick-alpine-theme.scss" I noticed lack of files "mixins.scss" and "variables.scss" in the "sass" folder which it tries to import.
As a workaround I can import styles from the "src" folder 
`@import "node_modules/slickgrid/src/styles/slick-alpine-theme.scss"`
But I'm not sure whether it's correct.

![image](https://github.com/6pac/SlickGrid/assets/60598547/a388cf40-9174-42e2-8d2c-36a9b769fda6)

### Reproduction

npm install slickgrid
in "index.scss" file try to import styles 
@import 'node_modules/slickgrid/dist/styles/sass/slick-alpine-theme.scss';

### Which Framework are you using?

React

### Environment Info

```shell
| Executable          | Version |
| ------------------- | ------- |
| React               | 17.0.2  |
| slickgrid           |  5.8.2  |
| TypeScript          |  4.7.4  |
| Windows             |   10    |
| Node.js             | 18.20.0 |
| npm                 | 10.5.0  |
```

### Validations

- [X] Follow our [Code of Conduct](https://github.com/6pac/SlickGrid/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [Wikis](https://github.com/6pac/SlickGrid/wiki).
- [X] Check that there isn't [already an issue](https://github.com/6pac/SlickGrid/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/6pac/SlickGrid/discussions).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by ghiscoding** [comment]
ah yeah that's a bug, I'm not sure why it doesn't copy all files, it should include the 2 files starting with underscore as well. It works in my other repo, 

https://github.com/6pac/SlickGrid/blob/13f69c4e569e69354fbb379ffa673ade29505173/package.json#L65

**Comment by ghiscoding** [comment]
should be fixed in v5.8.3

**Comment by AlexandrChazov** [comment]
Thanks a lot for fixing! It works now.
